src/components/drawer/Bag.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from 'react'
+import Box from '@mui/material/Box'
+import Drawer from '@mui/material/Drawer'
+import Button from '@mui/material/Button'
+import List from '@mui/material/List'
+import Divider from '@mui/material/Divider'
+import ListItem from '@mui/material/ListItem'
+import ListItemIcon from '@mui/material/ListItemIcon'
+import ListItemText from '@mui/material/ListItemText'
+import Typography from '@mui/material/Typography'
+import InboxIcon from '@mui/icons-material/MoveToInbox'
+import MailIcon from '@mui/icons-material/Mail'
+import LocalMallIcon from '@mui/icons-material/LocalMall'
+
+import CompsStyledBadge from '@/components/layouts/navbar/Badge'
+
+export default function CompsDrawerBag() {
+  const [openBag, setOpenBag] = useState(false)
+
+  const list = () => (
+    <Box
+      sx={{ width: 250 }}
+      role="presentation"
+      // onClick={() => setOpenBag(false)}
+    >
+      <Typography
+        align="center"
+        variant="h6"
+        component="div"
+        sx={{ flexGrow: 1, my: 3 }}
+      >BAG
+      </Typography>
+      <Typography
+        align="center"
+        variant="h6"
+        component="div"
+        sx={{ flexGrow: 1, mt: 4, mb: 2 }}
+      >YOUR PICKUP ORDER
+      </Typography>
+      <Divider />
+      <List>
+        {['Item 1', 'Item 2', 'Item 3', 'Item 4'].map((text, index) => (
```

Comment:
what are these for?

Reply:
{['Item 1', 'Item 2', 'Item 3', 'Item 4'].map((text, index) => (

^ is just a placeholder